packages/compass-import-export/test/docs/all-bson-types.js

```diff
@@ -10,47 +10,51 @@ import {
   MaxKey,
   MinKey,
   ObjectId,
-  Symbol as BSONSymbol,
+  //Symbol as BSONSymbol,
   Timestamp,
   UUID,
 } from 'bson';
 
 export default [
   {
+    _id: new ObjectId('642d766b7300158b1f22e972'),
     double: new Double(1.2), // Double, 1, double
     string: 'Hello, world!', // String, 2, string
     object: { key: 'value' }, // Object, 3, object
     array: [1, 2, 3], // Array, 4, array
+    // TODO: this is not being formatted in a way that gets seen by the parser. comes out as blank/null somehow
```

Comment:
Is this referring to the binData?

Reply:
Sorry old comment. The thing I was going on about in #dev-tools. Fixed now.